migrations/2024-09-25-113851_increase_connector_transaction_id_length_in_payment_and_refund/up.sql

```diff
@@ -0,0 +1,11 @@
+ALTER TABLE payment_attempt
+ADD COLUMN IF NOT EXISTS connector_transaction_data VARCHAR(512);
+
+ALTER TABLE refund
+ADD COLUMN IF NOT EXISTS connector_refund_data VARCHAR(512);
+
+ALTER TABLE refund
```

Comment:
why do we need to store this? isn't this already stored in payment attempt?

Reply:
Capture and Refund IDs for WP follows the same format. These IDs can differ across different resources (payments, refunds, captures) - they don't for WP, but better to store both the data and hashed values in the tables.